timeout_60mins_playbook.yaml

```diff
@@ -0,0 +1,6 @@
+---
+- name: Do nothing for 60 minutes (or more)
```

Comment:
This seems awfully long for a test context, can it be shortened down to the lowest possible interval for the test? 5m? 1m even?

Reply:
There was a customer BZ --> After running playbook for 60 mins or more, the time shown on the Service is something `01:65:02`.